### DISCUSSION TITLE
data limitation

### DISCUSSION THREAD

**Discussion by StepAll** [vega/vega-lite#7916] [Q&A] [answered]
Hi, guys!
First of all, thanks a  lot for a great job! Love the product.
Working with vega-lite and  80000+rows-dataset I've run into data limitation in 10000 rows. 
Is it done intentionally? 
I haven't found any information about it in the documentation.

**Comment by domoritz** [comment, marked as answer]
I am not aware of any explicit size limitation. At that scale, we may be limited by memory restrictions of the browser and excessive runtime. 

Any limitations would originate in Vega since Vega-Lite doesn't actually render the charts.

**Reply by StepAll** [reply]
hmm.. 
yeah, really there is a limit in Vega
https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#:~:text=the%20refresh%20interval.-,index.max_result_window,-The%20maximum%20value
I wonder if it can be increased in Deneb pbi app?

**Reply by domoritz** [reply]
This is not the right document. Elastic uses Vega but it's not the official documentation for Vega. @nyurik can help with Elastic questions.

**Reply by nyurik** [reply]
@StepAll yes, this is an elasticsearch query limitation - it returns max of 10,000 data points, and it has nothing to do with Vega.

**Reply by StepAll** [reply]
@nyurik thanks, 
so, finally,  this limitation imposed by the application, in my case deneb-pbi-viz  https://deneb-viz.github.io/ 
is it right?

**Reply by draftbin** [reply]
that's correct, Deneb limits to 10,000 rows but you cant override it in visulations pane under formatting. https://deneb.guide/docs/dataset
